src/main/java/games/strategy/engine/ClientContext.java

```diff
@@ -50,6 +54,8 @@
   private final BattleCalcSettings battleCalcSettings;
   private final BattleOptionsSettings battleOptionsSettings;
 
+  public static final DownloadCoordinator DOWNLOAD_COORDINATOR = new DownloadCoordinator();
```

Comment:
It looks like all the other services managed by `ClientContext` are instance fields.  I don't see a precedent for using a static field.  @DanVanAtta, since you were here recently, can you comment on what is the idiomatic way for adding services to `ClientContext`?